content/en/docs/how-tos/use-registries-in-build-farm.md

```diff
@@ -0,0 +1,118 @@
+---
+title: "Use Registries in Build Farm"
+date: 2020-12-15T16:08:01-04:00
+draft: false
+---
+
+The follow table shows the public DNS of each registry in build farm:
+
+| cluster | registry                                             | note                        |
+|---------|------------------------------------------------------|-----------------------------|
+| api.ci  | registry.svc.ci.openshift.org                        |                             |
+| app.ci  | registry.ci.openshift.org                            | the central CI registry     |
+| build01 | registry.build01.ci.openshift.org                    |                             |
+| build02 | registry.build02.ci.openshift.org                    |                             |
+| vsphere | registry.apps.build01-us-west-2.vmc.ci.openshift.org | only open to vsphere admins |
+
+# Login as a github user:
+
+* Ensure oc-cli has logged into the cluster: Copy the login command from [the console](/docs/getting-started/useful-links/#clusters).
+* Use oc token to log into the registry, e.g.,  `registry.ci.openshift.org`:
+
+```bash
+$ podman login -u $(oc whoami) -p $(oc whoami -t) --authfile auth.json registry.ci.openshift.org
```

Comment:
why the `authfile` parameter, is that really needed? Also, I would use `docker` as the command here as that is more standard but mention in the beginning that it is the same for `podman`

Reply:
Updated podman on centos8
```
podman -v
podman version 2.0.5
```

I cannot tell podman to use `${HOME}/.docker/config.json` for storing tokens, except using `--authfile`. Also tried $REGISTRY_AUTH_FILE, no luck. The default path is `/run/containers/0/auth.json`.